### PR TITLE
Restructure docu sections

### PR DESCRIPTION
**Pull request by bgeuken**
Main changes of this PR are

* Move adding repository documentation to 'Manage repositories' section.
* Fix a few typos.
* Make docu for adding repositories generic (by not mentioning Factory) and rephrased some parts.
* Refer from link packages walkthrough (section 1.4) to add repository section.
* Add a note element for expert mode.
* Itemize instruction for creating a package link. 

For the itemization I was able to make a screenshot. The rest are mainly text changes, and moving sections around:

before:
itemized_before

after:
itemized_after

### CODE REVIEW

**Comment by bgeuken** [comment]
@adrianschroeter Please review.

Note that this doesn't change the DoD docu yet. Except moving it to a different section.

File: xml/obs_best_practice_webui_usage.xml [hunk 1]
```diff
@@ -155,10 +155,24 @@
    Now let's create a link to a package and add a repository to build
    against. A link is basically a pointer to sources of an already existing
    package in another project. By *repository*, we mean a build target -
```
Comment (adrianschroeter) [inline xml/obs_best_practice_webui_usage.xml]: in another project -> at another location ?

There are also links pointing to the same project...

File: xml/obs_best_practice_webui_usage.xml [hunk 2]
```diff
@@ -155,10 +155,24 @@
...
-   like Trunk/Fedora_12/openSUSE_11.4. Let's follow these steps: 1. Add link
-   to the existing package. 2. Add repository / build target. 3. Enable the
-   build (optional, if not already enabled). 4. Observe the build on the
-   monitor page 5. Look at package's page
+   like Trunk/Fedora_12/openSUSE_11.4. Let's follow these steps:
```
Comment (adrianschroeter) [inline xml/obs_best_practice_webui_usage.xml]: By "repository" we mean container of built binary packages like Debian_8 or openSUSE_13.2 .

File: xml/obs_best_practice_webui_usage.xml [hunk 3]
```diff
@@ -155,10 +155,24 @@
...
+     <orderedlist>
+       <listitem>
+         <para>Add link to the existing package.</para>
+       </listitem>
+       <listitem>
+         <para>Add repository / build target.</para>
```
Comment (adrianschroeter) [inline xml/obs_best_practice_webui_usage.xml]: Can we just drop "build target" here? We have no such terminology IIRC.

File: xml/obs_best_practice_webui_usage.xml [hunk 4]
```diff
@@ -155,10 +155,24 @@
...
+       </listitem>
+       <listitem>
+         <para>Enable the build (optional, if not already enabled).</para>
```
Comment (adrianschroeter) [inline xml/obs_best_practice_webui_usage.xml]: that is the default, I would just skip it

File: xml/obs_best_practice_webui_usage.xml (hunk 2 above)
Comment (bgeuken) [inline xml/obs_best_practice_webui_usage.xml]: I replaced the sentence

File: xml/obs_best_practice_webui_usage.xml (hunk 3 above)
Comment (bgeuken) [inline xml/obs_best_practice_webui_usage.xml]: Ok

File: xml/obs_best_practice_webui_usage.xml (hunk 4 above)
Comment (bgeuken) [inline xml/obs_best_practice_webui_usage.xml]: I removed it now

**Pull request merged**